app/views/search/search.html.erb

```diff
@@ -0,0 +1,180 @@
+<%= provide(:title, "Search") %>
+<script type='text/javascript' src='https://maps.googleapis.com/maps/api/js?key=AIzaSyANRR2kNcJ85_61PYyROVKQrRjRtqhpEUs&callback=initialize'></script>
```

Comment:
please fix this error regarding to Google Maps API Key:
![image](https://user-images.githubusercontent.com/73863408/134465198-34645fad-2728-42cc-aa51-82de97c39594.png)
![image](https://user-images.githubusercontent.com/73863408/134465238-d71b550c-3e52-4d80-a0cb-94e48294d38d.png)

To view this error in your machine you can right click on any empty space in the browser then select "Inspect" or "Inspect Elements"